zrml/swaps/src/migrations.rs

```diff
@@ -0,0 +1,110 @@
+pub mod _0_1_1 {
```

Comment:
I'd give this an more descriptive name additionally, such we know which migration did what in the future. For example: `pub mod _0_1_1_add_pool_status`

Reply:
Nice